test/basic-images.test

```diff
@@ -339,6 +339,14 @@ test_expect_success mkfs_jffs2 "jffs2" "
 	md5sum -c '${testdir}/jffs2.md5'
 "
 
+exec_test_set_prereq mkfs.f2fs
+exec_test_set_prereq sload.f2fs
+exec_test_set_prereq fsck.f2fs
+test_expect_success mkfs_f2fs,sload_f2fs "f2fs" "
```

Comment:
You need to add fsck_f2fs here to. The line above just defines a possible requirement.